azure-devops/create-vmss.ps1

```diff
@@ -103,7 +103,7 @@ function New-Password {
 
   # This 64-character alphabet generates 6 bits of entropy per character.
   # The power-of-2 alphabet size allows us to select a character by masking a random Byte with bitwise-AND.
-  $alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789_-"
+  $alphabet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789_-'

```

Comment:
Is this just a consistency change?

Reply:
Oh I see:
> Use single quotes when we don't need expansion

I thought different quotes carried no semantic, like JavaScript

Reply:
Yep, there is a difference in PowerShell:
```
PS C:\Temp> $cats = "KITTENS"
PS C:\Temp> '$cats are cute'
$cats are cute
PS C:\Temp> "$cats are cute"
KITTENS are cute
```